src/features/firstLogin/tutorials/components/GenericCard.tsx

```diff
@@ -80,15 +80,21 @@ export const GenericCard: FunctionComponent<CardProps> = (props: CardProps) => {
       <StyledBody>{props.text}</StyledBody>
       <Spacer.Flex flex={2} />
       <Animatable.View ref={animatedButtonRef}>
-        {props.activeIndex === props.index && (
+        {props.activeIndex === props.index ? (
           <ButtonPrimary title={props.buttonText} onPress={props.buttonCallback} />
+        ) : (
+          <InvisibleButtonHeight testID="no-button" />
```

Comment:
j'ai eu du mal en lisant les test au niveau compréhension. Je sais pas si le naming est foufou, mais c'est déjà mieux :
```suggestion
          <InvisibleButtonHeight testID="invisible-button-height" />
```